src/python/pants/core/util_rules/environments.py

```diff
@@ -514,6 +557,19 @@ def executable_search_path_cache_scope(
             else ProcessCacheScope.PER_RESTART_SUCCESSFUL
         )
 
+    def sandbox_base_path(self) -> str:
+        if self.val and self.val.has_field(LocalWorkspaceCompatiblePlatformsField):
+            return "{chroot}"
+        else:
+            return ""
+
+    @property
+    def default_cache_scope(self) -> ProcessCacheScope:
+        if self.val and self.val.has_field(LocalWorkspaceCompatiblePlatformsField):
+            return ProcessCacheScope.PER_SESSION
+        else:
+            return ProcessCacheScope.SUCCESSFUL
```

Comment:
Added this method as a way for an environment to specify what the default cache scope is. 

@benjyw: Is this what you were thinking with your suggestion?

If so, this maybe should be pulled out and all applicable use sites updated.

Reply:
Yeah, this makes sense, but see my further question above about what "caching" even means in this context.